Gemfile.lock

```diff
@@ -174,7 +174,7 @@ PATH
     decidim-proposals (0.31.0.dev)
       decidim-comments (= 0.31.0.dev)
       decidim-core (= 0.31.0.dev)
-      doc2text (~> 0.4.7)
+      doc2text (~> 0.4.0)
```

Comment:
This is 0.4.0 instead of 0.4.8, right?

Reply:
YES. That is a "zero", meaning we can install `0.4.6`, `0.4.7`, `0.4.8` . But I am submitting an update, to make it clearer.